### PR TITLE
[Helm] Load environment variables from secrets

### PR DESCRIPTION
**Pull request by nlamirault**
Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

### CODE REVIEW

File: operations/phlare/helm/phlare/templates/deployments-statefulsets.yaml [hunk 1]
```diff
@@ -65,6 +65,10 @@ spec:
               value: {{ $value | quote }}
            {{- end }}
           {{- end }}
+          {{- with .Values.extraEnvFrom }}
```
Comment (simonswine) [inline operations/phlare/helm/phlare/templates/deployments-statefulsets.yaml:68]: We allow a couple of overrides to be happening and hence you need to access the values like this:

```suggestion
          {{- with .$values.extraEnvFrom }}
```

**Review by simonswine — COMMENTED** [review]
Thanks for your contribution! 

I think the way it is, it won't work as you won't access the correct values, see my specific comment.

Other than that: 
* Could you please rebase with the latest main (to fix the broken tests)?
* If possible run `make helm/check` and commit the changed file (we really should have a CI check for that #396 )

**Review by simonswine — APPROVED** [review]
Perfect! That's ready to go. Thanks for the contribution

**Comment by nlamirault** [comment]
@simonswine any news to release a new chart version with this feature ?

**Comment by nlamirault** [comment]
Thanks @simonswine

**Pull request merged**